### ISSUE TITLE
[YSQL] basic pg_dump support: TRANSACTION READ ONLY mode not supported yet

### ISSUE THREAD

**Issue by kmuthukk** [YugaByte/yugabyte-db#876]
```
00:12 $ ./postgres/bin/pg_dump -h 127.0.0.1 -p 5433 -U postgres -t stock_market demo

pg_dump: [archiver (db)] query failed: ERROR:  TRANSACTION READ ONLY mode not supported yet
LINE 1: SET TRANSACTION ISOLATION LEVEL REPEATABLE READ, READ ONLY
                                                         ^
HINT:  Please contact YugaByte for its release schedule.
pg_dump: [archiver (db)] query was: SET TRANSACTION ISOLATION LEVEL REPEATABLE READ, READ ONLY
```

**Comment by kmuthukk** [comment]
Basically, I want to see the "create table" statement for a given table, but 

```
hello=# \d stock_market
                    Table "public.stock_market"
    Column     |       Type       | Collation | Nullable | Default
---------------+------------------+-----------+----------+---------
 stock_symbol  | text             |           |          |
 ts            | text             |           |          |
 current_price | double precision |           |          |
```
doesn't list the primary key columns.


So, I am trying instead to do something like this:
```
00:17 $ ./postgres/bin/pg_dump -h 127.0.0.1 -p 5433 -U postgres hello -t stockmarket --schema-only
```

but get this error:
```
pg_dump: [archiver (db)] query failed: ERROR:  TRANSACTION READ ONLY mode not supported yet
LINE 1: SET TRANSACTION ISOLATION LEVEL REPEATABLE READ, READ ONLY
                                                         ^
HINT:  Please contact YugaByte for its release schedule.
pg_dump: [archiver (db)] query was: SET TRANSACTION ISOLATION LEVEL REPEATABLE READ, READ ONLY 
```

**Comment by ravimurthy** [comment]
Duplicate of https://github.com/YugaByte/yugabyte-db/issues/868

**Comment by m-iancu** [comment]
Fixed by @ndeodhar in f5cf9b9. See also duplicate #868.
